dsl/src/test/scala/asobu/dsl/SyntaxSpec.scala

```diff
@@ -175,17 +201,17 @@ class SyntaxSpec extends PlaySpecification {
 
     case class SessionInfo(sessionId: String)
 
-    "with AuthExtractor" >> { implicit ev: ExecutionEnv ⇒
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method name does not match the regular expression '^([a-z][A-Za-z0-9]*|<<|>>>|>|==|!=|<|<=|>|>=|\||&|\^|\+|-|\*|\/|%|\|\||&&|\+\+|--|\+=|-=)$'](https://www.codacy.com/app/iHeartRadio/asobu/file/2287259465/issues/source?bid=3037259&fileBranchId=3178877#l204)